test/powershell/Modules/Microsoft.PowerShell.Core/ModuleConstraint.Tests.ps1

```diff
@@ -83,6 +83,39 @@ function Invoke-ImportModule
     return Import-Module @cmdArgs
 }
 
+function Assert-ModuleIsCorrect
+{
+    param(
+        $Module,
+        [string]$Name,
```

Comment:
Yes I think that makes sense 👍

Reply:
Worth noting that the `$moduleName` one relies on dynamic scope. I kept the other parameters, since I think it's less confusing. But it's not quite consistent with `$moduleName`.